packages/loader/container-loader/src/container.ts

```diff
@@ -2333,6 +2333,10 @@ export class Container
 		);
 	}
 
+	/**
+	 * Processes incoming delta messages
```

Comment:
nit: I would not call it delta message. Just message should be enough.
Also, maybe add somewhere that the messages are coming from other clients (not sent by this client) hence remote.

Reply:
I am thoroughly confused now :D. I thought the messages are coming from server which could be from other clients or could even be the ones that were created by current client - after they are sequenced. Is that not correct?

I wanted to explicitly specify delta messages, because it represents connection to the delta service, and not storage relates api responses.

Reply:
My bad. I assumed from the name that it only processes remote messages after filtering local ones.

Api responses are not called messages so calling them both delta and messages are redundant. Basically, we have 3 terms for the same thing - op, message and delta. Anyway, it was a nit comment, so feel free to ignore it :-).